apps/astarte_data_updater_plant/lib/astarte_data_updater_plant/triggers_handler.ex

```diff
@@ -153,27 +178,39 @@ defmodule Astarte.DataUpdaterPlant.TriggersHandler do
         timestamp
       )
       when is_list(targets) do
-    execute_all_ok(targets, fn target ->
+    execute_all_ok(targets, fn {target, policy} ->
       interface_added(
         target,
         realm,
         device_id,
         interface,
         major_version,
         minor_version,
-        timestamp
+        timestamp,
+        policy
       ) == :ok
     end)
   end
 
+  @spec interface_added(
+          Astarte.Core.Triggers.SimpleTriggersProtobuf.AMQPTriggerTarget.t(),
+          any,
+          any,
+          any,
+          any,
+          any,
+          any,
+          any
+        ) :: :ok
```

Comment:
Do we need this?

Reply:
![image](https://user-images.githubusercontent.com/27413985/200792263-448136da-7e3c-4453-ab8e-9d18c4a2e5dc.png)